### PR TITLE
feat: store bench

### PR DESCRIPTION
**Pull request by dbcfd**
Use criterion to benchmark store inserts

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -201,3 +203,6 @@ authors = [
 ]
 license = "Apache-2.0/MIT"
 repository = "https://github.com/3box/rust-ceramic"
+
+[patch.crates-io]
+criterion2 = { git = "https://github.com/dbcfd/criterion2.rs.git", branch = "feat/async-setup" }
```
Comment (dav1do) [inline Cargo.toml:208]: why do we need a fork?

File: Cargo.toml [hunk 2]
```diff
@@ -61,7 +61,7 @@ clap = { version = "4", features = ["derive", "env"] }
 clap_mangen = "0.2.2"
 console = { version = "0.15", default-features = false }
 console-subscriber = "0.1.7"
-criterion = "0.4"
+criterion2 = "0.7.0"
```
Comment (dav1do) [inline Cargo.toml:64]: why criterion2 vs criterion?

File: Cargo.toml (hunk 1 above)
Comment (dbcfd) [reply Cargo.toml:208]: Needed the PR I put up merged and released.

File: Cargo.toml (hunk 2 above)
Comment (dbcfd) [reply Cargo.toml:64]: criterion no longer seems to be actively  maintained

File: Cargo.toml (hunk 1 above)
Comment (dav1do) [reply Cargo.toml:208]: It looks like it's been released? Mind updating it when you resolve the conflicts?

**Review by dav1do — APPROVED** [review]

**Pull request merged**